roles/cdi/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+cdi_namespace: "{{ cdi_image_namespace | default('golden-images') }}"
+action: "provision"
+cdi_template_dir: "{{ role_path }}/templates"
```

Comment:
I've learned that roles take templates from role_path/templates by default so you can drop this variable.